README.md

```diff
@@ -1,39 +1,55 @@
 ![Logo](imgs/joligan.svg)
 
+<h1 align="center">Image-to-Image Translation for Domain Adaptation</h1>
 
-JoliGAN is an implementation of an unpaired image to image translation. It uses cycle consistency such as CycleGAN but it allows the use of :
-- more generator architectures such as styleGAN2 decoder / mobile resnet, attention resnet (and a mobile version)
-- semanctic consistency 
-- new losses : out mask loss, w loss (for sty2 decoder)
- 
-JoliGAN also includes an implementation of contrastive unpaired translation (CUT) enhanced with our achitectures and losses.
+**JoliGAN** provides models and machine learning tools for unpaired image to image translation and domain adaptation.
+- Multiple models based on adversarial generation: [CycleGAN](https://arxiv.org/abs/1703.10593), [CyCADA](https://arxiv.org/abs/1711.03213), [CUT](https://arxiv.org/abs/2007.15651) and more
+- Many generator architectures such as styleGAN2 decoder / mobile resnet, attention resnet,...
+- Semantic consistency allowing to perform domain adaptation e.g. from synthetic to real-like data
+
+---
+
+## Use cases
+
+- Image to image translation while preserving semantic
+
+_glasses 2 no glasses, explicative schemas, results_
+
+- Image to image translation to cope with scarce data
+
+_eye gaze or another dataset, schemas, results_
```

Comment:
These are placeholders, I can comment them out or add an illustration image for the moment

Reply:
Maybe you can comment them out to merge this PR. We will add illustration images and explanations in a further PR.

Reply:
Unclear as well IMO.